extensions/positron-r/src/runtime-manager.ts

```diff
@@ -32,8 +35,20 @@ export class RRuntimeManager implements positron.LanguageRuntimeManager {
 	}
 
 	async recommendedWorkspaceRuntime(): Promise<positron.LanguageRuntimeMetadata | undefined> {
-		// TODO: If the workspace contains an R project, we could recommend an
-		// R runtime from e.g. the `DESCRIPTION` file or an renv lockfile.
+		// If the default interpreter path is set and the path exists on the filesystem,
+		// recommend it with implicit startup behavior.
+		const defaultInterpreterPath = getDefaultInterpreterPath();
+		if (defaultInterpreterPath) {
```

Comment:
Given the other existence checks, is it possible for this to be `false`? Mostly just curious.

Reply:
We don't preset a default interpreter path, so if the user hasn't set this, it will be `undefined`. We won't have done any "does this path actually exist?" checks at this point, just checks for path format like resolving the `~` if any and checking that the path looks like an absolute path.